src/Test.Plugins/ContextGeneratingHelper.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.using System;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.WorkItems;
+using Microsoft.CodeAnalysis.Test.Plugins;
+using Xunit;
+
+namespace Test.Plugins
+{
+    public class ContextGeneratingHelper
+    {
+        [Fact]
+        public void GenerateSampleContextToTestPlugin()
```

Comment:
>GenerateSampleContextToTestPlugin [](start = 20, length = 33)

This shouldn't be a test that's run; it should be a static method that's called from the "test setup" or "assembly setup" methods of your test classes.